src/controllers/serviceaccounts.go

```diff
@@ -96,3 +125,54 @@ func (c *serviceAccountController) getServiceAccountToken(serviceAccount *corev1
 
 	return "", customerrors.NewValidationError(fmt.Sprintf(ErrNoTokenFound, serviceAccount.Name))
 }
+
+// CreateServiceAccount creates a new ServiceAccount with the given name and namespace.
+func (c *serviceAccountController) CreateServiceAccount(name, namespace string) (types.MessageResponse, error) {
+	serviceAccount := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels: map[string]string{
+				utils.ManagedLabel: "true",
+			},
+		},
+	}
+	_, err := c.client.CoreV1().ServiceAccounts(namespace).Create(c.ctx, serviceAccount, metav1.CreateOptions{})
+	if err != nil {
+		return types.MessageResponse{}, customerrors.NewAPIError(fmt.Sprintf(ErrCouldNotCreateServiceAccount, name, namespace), err)
+	}
+	return types.OkResponse, nil
```

Comment:
I'd rather we return the name of the of the `serviceAccount` as a response, similar to what we do with `namespaces` 

https://github.com/dana-team/platform-backend/blob/50e4edbe3d2d405e7f38cfa62d983d2bf5e1cb2d/src/controllers/namespace.go#L98